bindings/cpp/result_entry.cpp

```diff
@@ -354,14 +454,81 @@ void iterator_result_container::append(const dnet_iterator_response *response)
 //* Sort container by (key, timestamp) tuple
 void iterator_result_container::sort()
 {
-	int err;
-
 	if (m_sorted == true)
 		return;
 
-	err = dnet_iterator_response_container_sort(m_fd, m_write_position);
-	if (err != 0)
-		throw_error(err, "sort failed");
+	if (m_write_position % sizeof(struct dnet_iterator_response) != 0)
+		throw_error(-EINVAL, "invalid container size");
+
+	int err;
+	std::vector<std::shared_ptr<iterator_result_chunk> > chunks;
```

Comment:
1 Tb / 500 Mb ~= 2000 - it is still very small
Do we really expect 1 Tb iterator results in near future?